auto_nag/scripts/regression_set_status_flags.py

```diff
@@ -168,8 +168,9 @@ def get_status_changes(self, bugs):
             for v in esr_versions:
                 info.setdefault("esr", {})
                 flag = utils.get_flag(v, "status", "esr")
-                info["esr"][f"esr{v}"] = info[flag]
-                if info[flag] != "---":
+                flag_status = info.get(flag)
```

Comment:
Maybe we should explicitly `continue` when the flag is not present, instead of relying on the `if flag_status != "---":` check so when somebody touches this in the future they will know

Reply:
I agree; done in a89dd3f6ddff8ab92402a3467410df5d4cb48984.